tests/storage/lvm_test.py

```diff
@@ -43,6 +43,33 @@
 from . marks import requires_root
 
 
+EXPECTED_LVM_CONFIG = """
+devices {
+ preferred_names=["^/dev/mapper/"]
+ ignore_suspended_devices=1
+ write_cache_state=0
+ disable_after_error_count=3
+ %(filter)s
+ hints="none"
+ obtain_device_list_from_udev=0
+}
+global {
+ prioritise_write_locks=1
+ wait_for_locks=1
+ use_lvmpolld=1
+}
+backup {
+ retain_min=50
+ retain_days=0
+}
+"""
```

Comment:
This looks like a duplicate of the template in lvm.py. What is the difference?

Before this change, we have simple text that does not need any processing. I don't
see need to duplicate the code of lvm.py for testing the code.

It is ok to keep static text - once for the case when we don't have a filter,
and one for the case we have a filter.